app/models/ott/CategorisationInfo.scala

```diff
@@ -24,12 +24,17 @@ final case class CategorisationInfo(
   commodityCode: String,
   categoryAssessments: Seq[CategoryAssessment],
   measurementUnit: Option[String],
-  descendantCount: Int
-)
+  descendantCount: Int,
+  originalCommodityCode: String = ""
```

Comment:
Probably makes this messier as you'd have to map it below but I guess more functional(?) would be an Option[String] rather than defaulting to empty

Reply:
I sort of agree, the issue is that optional causes some of the issues liandra commented - and having it non-optional without it initialised means I'd need to refactor a lot of tests that are unrelated. i'm unfortunately inclined to keep it for now